src/kernels/jupyter/launcher/liveshare/serverCache.ts

```diff
@@ -110,7 +110,6 @@ export class ServerCache implements IAsyncDisposable {
         }
         return {
             serverId: options.serverId,
-            uri: options.uri,
```

Comment:
I think this is good now, we don't really need to pass the Uri around as this also contains sensitive information & we don't accidentally log this anymore.